dapps/marketplace/src/hoc/withGrowthCampaign.js

```diff
@@ -7,12 +7,15 @@ import enrollmentStatusQuery from 'queries/EnrollmentStatus'
 import profileQuery from 'queries/Profile'
 import allCampaignsQuery from 'queries/AllGrowthCampaigns'
 
-function withGrowthCampaign(WrappedComponent) {
+function withGrowthCampaign(
+  WrappedComponent,
+  { useCache, queryEvenIfNotEnrolled, suppressErrors } = {}
```

Comment:
maybe we can pass in a fetchPolicy prop and default it to 'network-only'?

Reply:
👍